charts/rstudio-pm/NEWS.md

```diff
@@ -1,3 +1,9 @@
+# 0.3.11
+
+- Add Package Manager configuration values
+  - `labels`, `sidecar`, `affinity`, `nodeSelector`, `tolerations`, `priorityClassName`
+  - close [#206](https://github.com/rstudio/helm/issues/206)
```

Comment:
Nit: I think this could all be one line.

```suggestion
- Add configuration values for the pod's `labels`, `sidecar`, `affinity`, `nodeSelector`, `tolerations`, and `priorityClassName` ([#206](https://github.com/rstudio/helm/issues/206)).
```